install/ossm-installing-openshift-service-mesh.adoc

```diff
@@ -11,6 +11,8 @@ Installing {ocp-short-name} {SMProductShortName} consists of three main tasks: i
 [WARNING]
 ====
 Before migrating from {SMProduct} 2, make sure you are not running {SMProduct} 3 and {SMProduct} 2 in the same cluster, because it causes conflicts unless configured correctly. To migrate from {SMProduct} 2, see xref:../migrating/checklists/ossm-migrating-read-me.adoc#ossm-migrating-read-me[Migrating from {SMProduct} 2.6].
+
+For clusters without existing {SMProduct} instances, ensure proper scope configuration to prevent conflicts between {istio} control planes, as {SMProductShortName} is deployed cluster-wide. For clusters with {SMProduct} 3 or later, see xref:../install/ossm-deploying-multiple-service-meshes-on-single-clusterc#ossm-deploying-multiple-service-meshes-on-single-cluster[Deploying multiple service meshes on a single cluster].
```

Comment:
I'm not sure if I understand. How can we assure proper scope configuration for clusters without existing OSSM instances?

Reply:
Hi @FilipB! I have moved the second part of the WARNING before the installing the operator procedure. I think it will make more sense there. I have reworded it too. PTAL. Thanks!